---
layout: page
title: "Current research"
subheadline: ""
teaser: "I am a Senior Bioinformatician at the U. of Washington, where I provide data analysis expertise in next-generation sequencing and other high-throughput molecular data across a variety of projects relevant to human health and disease."
permalink: "/current/"
header:
    image_fullwidth: "barplot.png"
---

## Public project repos
<br>
[Eosinophils and Anti-IL-5 therapy impact dendritic cells in asthma ][P259]  
[TOLLIP, lipids, and innate immunity during prolonged *M. tuberculosis* infection][JS20]

## Transcriptional responses
<br>
Upwards of 20% of the human transcriptome changes in response to infection. These changes correspond with our ability to clear infections as well as with symptom severity and treatment efficacy. Thus, transcriptome profiling can provide a wealth of data useful in preventing and treating human disease. 

Utilizing **linear modeling, gene set enrichment analysis (GSEA), and network analysis**, I leverage RNA-seq data sets to determine how immune cells respond to infection in the context of diverse human populations, co-morbidities, and new therapies. Specifically, working with [Dr. Tom Hawn][hawn], I investigate how monocytes from an established, tuberculosis-resistant cohort as well as how permissive macrophages (M2) from healthy donors respond to *M. tuberculosis* infection. In addition, with [Dr. Matt Altman][altman], I research interactions between asthma, respiratory viruses, and preventative therapeutics, particularly in children. This includes focused *in vitro* immune cell co-cultures as well as *in vivo* drug effects.

## Epigenetic and genetic programming
<br>
Transcriptional responses vary between individuals due to many epigenetic, genetic, and environmental factors. Markers related to gene expression and disease outcomes can give insight into the molecular mechanisms of disease and provide targets for prediction and treatment. Toward this goal with Dr. Hawn, I assess methylation (EPIC), chromatin availability (ATAC-seq), and single nucleotide polymorphisms (SNP, MEGA<sup>EX</sup>) in tuberculosis-resistant, latent tuberculosis infected (LTBI), and unexposed populations. I integrate these large data sets using **principle component analysis (PCA), partial least squares (PLS), and LASSO regression** as well as directly relate them to gene expression using **expression quantitative trait (eQT) analyses**.

## Lung - brain - body
<br>
The majority of my work focuses on DNA/RNA, the human lung, and immune cells important in that environment. However, through collaborations at UW and beyond, I continue to expand my data expertise and understanding of human health and disease. With Dr. Altman, [Dr. Melissa Rosenkranz][rosenkranz], and [Dr. Bill Busse][busse], I combine microbiology and psychology in a project mapping connections between the lung (RNA-seq) and brain function (fMRI). With Dr. Hawn and [Dr. Tomas Vaisar][vaisar], I look at proteins, lipids, and lipoproteins circulating throughout the body.

These and other collaborations are part of our on-going efforts to build bioinformatic capacity at UW South Lake Union and integrate robust, reproducible analysis throughout our department.

[altman]: https://aid.uw.edu/faculty/matthew-c-altman-md-mphil
[busse]: https://www.medicine.wisc.edu/people-search/people/staff/53/Busse_William
[hawn]: https://aid.uw.edu/faculty/thomas-r-hawn-md-phd
[shah]: https://aid.uw.edu/faculty/javeed-ali-shah-md
[rosenkranz]: https://psych.wisc.edu/staff/rosenkranz-melissa-a/
[vaisar]: https://uwmdi.org/faculty/tomas-vaisar/

[rstr]: https://doi.org/10.1038/s41577-018-0025-3
[P259]: https://github.com/altman-lab/P259_pDC_public
[JS20]: https://github.com/altman-lab/JS20.01
