---
layout: page
title: "Resume"
subheadline: ""
teaser: "Microbiology PhD with 10+ years experience applying data science and statistics to biological problems. Expertise in multi-omics from experimental design to publication. Skilled leader of teams with diverse backgrounds and varied training. Effective communicator with formal teaching experience. Specific skills include:"
permalink: "/short/"
header:
    image_fullwidth: "man.plot.png"
---

<table border="0" width="100%" class="center">
 <tr>
    <td width="35%"><b style="font-size:16px">Data</b></td>
    <td width="30%"><b style="font-size:16px">Software</b></td>
    <td width="35%"><b style="font-size:16px">Statistics</b></td>
 </tr>
 <tr>
    <td> <ul>
    <li>Transcriptomics: RNA-seq, scRNA-seq</li>
    <li>Genetics: MegaEX, lpWGS, whole exome</li>
    <li>Epigenetics: ATAC-seq, bisulfite seq</li>
    <li>Proteomics: MS/MS</li>
    <li>Microbiome: 16S, 18S, ITS, meta-transcriptomics</li>
    </ul> </td>
    
    <td> <ul>
    <li>AWS</li>
    <li>Git, GitHub</li>
    <li>R, Rmd, Shiny</li>
    <li>STAR, BWA, bowtie</li>
    <li>Seurat, STARsolo</li>
    <li>PLINK, vcftools</li>
    <li>mothur, QIIME</li>
    </ul> </td>
    
    <td> <ul>
    <li>Linear, additive, PLS, LASSO, latent regression</li>
    <li>Un/supervised machine learning</li>
    <li>PCA, NMDS, tSNE</li>
    <li>Sparse data imputation</li>
    <li>Data visualization</li>
    </ul> </td>
 </tr>
</table>

### Experience
<br>
__*07/2021 - present* --- Senior Bioinformatician, Allergy and Infectious Diseases, U. of Washington__

<ul><li>Lead a team of bioinformaticians in the analysis of ‘omics data for multiple research groups across the Dept of Medicine</li></ul>
<ul><li>Director of the Human Immunology Project Consortium ( [HIPC](https://immunespace.org/) ) Data Management and Analysis Core</li></ul>
<ul><li>Develop, standardize, and publish 'omics workflows in R packages and other software</li></ul>
<ul><li>Contribute to scientific grants and publications</li></ul>

__*07/2019 - 06/2021* --- Bioinformatician, Allergy and Infectious Diseases, U. of Washington__

<ul><li>Analyze human transcriptomic, epigenetic, and genetic datasets including quality assessment, statistical tests, and visualization</li></ul>

<ul><li>Mentor researchers and students as Founder and Chair of the Bioinformatics Group at South Lake Union ( [BIGslu](https://github.com/BIGslu) ) and Informatics for TB ( [i4TB](https://seatrac.uw.edu/training/i4TBworkinggroup) )</li></ul>

__*08/2017 - 06/2019* --- Post-doctoral research fellow, Microbiology & Immunology, U. of British Columbia__

<ul><li>Directed the Experiential Data Science for Undergraduate Cross-disciplinary Education ( [EDUCE](https://educe-ubc.github.io/) ) program</li></ul>
<ul><li>Designed and implemented data science curriculum in 7 courses</li></ul>
<ul><li> [ECOSCOPE](http://ecoscope.ubc.ca/) fellow coordinating and teaching data science workshops for industry and academic partners</li></ul>

__*07/2016 - 06/2017*	--- Post-doctoral research associate, Bacteriology/Sociology, U. of Wisconsin-Madison__
<ul><li>Investigated relationships between the human gut microbiome and longterm behaviors in the Wisconsin Longitudinal Study ( [WLS](https://www.ssc.wisc.edu/wlsresearch/) )</li></ul>
<ul><li>Navigated large, longitudinal survey databases </li></ul> 

__*08/2011 - 06/2016*	--- Graduate research assistant, Bacteriology, U. of Wisconsin-Madison__
<ul><li>Designed and implemented amplicon sequence analysis pipelines</li></ul>
<ul><li>Collaborated effectively with diverse international researchers resulting in first and corresponding author publications</li></ul>
<ul><li>Mentored high school, undergraduate, and graduate students </li></ul> 

__*08/2008 - 05/2011* --- Undergraduate research assistant, Biology, U. of Puget Sound__
<ul><li>Thesis: Investigating maltose metabolism in *Bdellovibrio bacteriovorus*</li></ul>  


### Education
<br>
2011 - 2016	--- Ph.D. Microbiology, U. of Wisconsin-Madison, Madison, WI. GPA: 4.00

2007 - 2011	--- B.S. Molecular and cellular biology, Minor mathematics, U. of Puget Sound, Tacoma, WA. GPA: 3.84

### Select publications
<br>
**Dill-McFarland KA**, Mitchell K, Batchu S, Segnitz RM, Benson B, Janczyk T, Cox MS, Mayanja-Kizza HA, Boom WH, Benchek P, Stein C, Hawn TR, Altman MC. 2023. kimma: Flexible linear mixed effects modeling with kinship for RNA-seq data. Bioinformatics. 39(5):btad279. doi: [10.1093/bioinformatics/btad279](https://doi.org/10.1093/bioinformatics/btad279) --- [GitHub](https://github.com/BIGslu/kimma)

Jackson DJ, Bacharier LB, Gergen PJ ... **Dill-McFarland KA** ... Altman MC, US NIAID Inner City Asthma Consortium. 2022. Mepolizumab for urban children with exacerbation-prone eosinophilic asthma in the USA (MUPPITS-2): a randomised, double-blind, placebo-controlled, parallel-group trial. Lancet 400(10351): 502-511 doi: [10.1016/S0140-6736(22)01198-9](https://doi.org/10.1016/s0140-6736(22)01198-9)

**Dill-McFarland KA**, Schwartz JT, Zhao H, Shao B, Fulkerson PC, Altman MC, Gill MA. 2022. Eosinophil-mediated suppression and Anti-IL-5 enhancement of plasmacytoid dendritic cell interferon responses in asthma. J Allergy Clin Immunol. Epub ahead of print. doi: [10.1016/j.jaci.2022.03.025](https://doi.org/10.1016/j.jaci.2022.03.025) --- [GitHub](https://github.com/altman-lab/P259_pDC_public)

[Full publication list](https://kdillmcfarland.github.io/pubs/)

### Notable accomplishments
<br>
2017, 2018, 2019, 2022 --- Session convener American Society for Microbiology (ASM) Microbe meeting

2016 --- Microbiome Digest’s [Best Microbiome Paper](https://microbiomedigest.com/2016/12/31/winners-of-the-best-microbiome-papers-of-2016/)

2016 --- Sigrid Leirmo Memorial Award for peer mentorship and support

2009, 2010 --- American Society for Microbiology Undergraduate Fellow

### Contact
<br>
University of Washington, Division of Allergy and Infectious Diseases  
kadm [at] uw.edu  
@kdillmcfarland
