---
layout: page
title: "Resume"
subheadline: ""
teaser: "Microbiology PhD with 9+ years experience in applying data science and statistics to biological problems. Skilled leader of teams with diverse backgrounds and varied training. Expertise in next-generation sequencing from experimental design to publication. Effective communicator with formal teaching experience. Specific skills include:"
permalink: "/short/"
header:
    image_fullwidth: "man.plot.png"
---

<table border="0" width="120%" class="center">
 <tr>
    <td width="40%"><b style="font-size:16px">Data</b></td>
    <td width="20%"><b style="font-size:16px">Software</b></td>
    <td width="40%"><b style="font-size:16px">Statistics</b></td>
 </tr>
 <tr>
    <td> <ul>
    <li>Transcriptomics: RNA-seq</li>
    <li>Genetics: MegaEX, Omni, whole exome</li>
    <li>Epigenetics: ATAC-seq, bisulfite seq</li>
    <li>Proteomics: MS/MS</li>
    <li>Microbiome: 16S, 18S, ITS, meta-transcriptomics</li>
    </ul> </td>
    
    <td> <ul>
    <li>AWS</li>
    <li>Git, GitHub</li>
    <li>R, Rmd, Shiny</li>
    <li>samtools, bedtools</li>
    <li>STAR, BWA, bowtie</li>
    <li>PLINK, vcftools</li>
    <li>mothur, QIIME</li>
    </ul> </td>
    
    <td> <ul>
    <li>Linear, PLS, LASSO, latent regression</li>
    <li>Un/supervised machine learning</li>
    <li>PCA, NMDS, tSNE</li>
    <li>Sparse data imputation</li>
    <li>Data visualization</li>
    </ul> </td>
 </tr>
</table>

### Experience
<br>
__*07/2019 - present* --- Bioinformatician, Allergy and Infectious Diseases, U. of Washington__

<ul><li>Analyze human transcriptomic, epigenetic, and genetic datasets including quality assessment, and statistical tests</li></ul>
<ul><li>Develop novel pipelines for integrated ‘omics analysis</li></ul>
<ul><li>Lead a team of 3 core bioinformaticians as well as mentor post-docs and students in related groups</li></ul>
<ul><li>Administrator AWS Organization (6 users)</li></ul>

__*04/2021 - present*	--- Genomic data specialist, San Mateo County Public Health Lab (part-time)__

<ul><li>Genome assembly, variant tracking, and outbreak assessment of COVID-19 clinical samples</li></ul>

__*08/2017 - 06/2019*	--- Post-doctoral research fellow, Microbiology & Immunology, U. of British Columbia__

<ul><li>Directed Experiential Data Science for Undergraduate Cross-disciplinary Education ([EDUCE](https://educe-ubc.github.io/)) including a team of 3 teaching assistants</li></ul>
<ul><li>Designed and implemented data science curriculum in 7 courses</li></ul>
<ul><li> [ECOSCOPE](http://ecoscope.ubc.ca/) fellow coordinating and teaching data science workshops for industry and academic partners</li></ul>

__*07/2016 - 06/2017*	--- Post-doctoral research associate, Bacteriology/Sociology, U. of Wisconsin-Madison__
<ul><li>Investigated relationships between the human gut microbiome and longterm behaviors in the Wisconsin Longitudinal Study ([WLS](https://www.ssc.wisc.edu/wlsresearch/))</li></ul>
<ul><li>Navigated large, longitudinal survey databases </li></ul> 

__*08/2011 - 06/2016*	--- Graduate research assistant, Bacteriology, U. of Wisconsin-Madison__
<ul><li>Designed and implemented amplicon sequence analysis pipelines</li></ul>
<ul><li>Collaborated effectively with diverse international researchers resulting in first and corresponding author publications</li></ul>
<ul><li>Mentored high school, undergraduate, and graduate students </li></ul> 

__*08/2008 - 05/2011* --- Undergraduate research assistant, Biology, U. of Puget Sound__
<ul><li>Thesis: Investigating maltose metabolism in *Bdellovibrio bacteriovorus*</li></ul>  


### Education
<br>
2011 - 2016	--- Ph.D. Microbiology, U. of Wisconsin-Madison, Madison, WI. GPA: 4.00

2007 - 2011	--- B.S. Molecular and cellular biology, Minor mathematics, U. of Puget Sound, Tacoma, WA. GPA: 3.84

### Select publications
<br>
**Dill-McFarland KA**\*, Tang Z\*, Kemis JH, Kerby RL, Chen G, Palloni A, Sorenson T, Rey FE†, Herd P†. 2019. Close social relationships correlate with human gut microbiota composition. Sci Rep 9: 703. doi: [10.1038/s41598-018-37298-9](https://www.nature.com/articles/s41598-018-37298-9) --- [GitHub](https://github.com/kdillmcfarland/WLS)

 **Dill-McFarland KA**, Weimer PJ, Breaker JD, Suen G. 2019. Diet influences early microbiota development in dairy calves without long-term impacts on milk production. Appl Environ Microbiol 85(2): e02141-18. doi: [10.1128/AEM.02141-18](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6328763/) --- [GitHub](https://github.com/kdillmcfarland/GS01)
 
**Dill-McFarland KA**, König SG, Mazel F, Oliver DC, McEwen LM, Hong KY, Hallam SJ. 2021. An integrated, modular approach to data science education in microbiology. PLoS Comput Biol 17(2): e1008661. doi: [10.1371/journal.pcbi.1008661](https://doi.org/10.1371/journal.pcbi.1008661) --- [GitHub](https://github.com/kdillmcfarland/EDUCE_desc)

\*co-first authors  
[Full publication list](https://kdillmcfarland.github.io/pubs/)

### Notable accomplishments
<br>
2017, 2018, 2019 --- Session moderator/convener at the American Society for Microbiology (ASM) Microbe meeting

2016 --- Microbiome Digest’s Best Microbiome Paper

2016 --- Sigrid Leirmo Memorial Award for peer mentorship and support

2009, 2010 --- American Society for Microbiology Undergraduate Fellow

### Contact
<br>
University of Washington, Division of Allergy and Infectious Diseases, Box 358061  
kadm [at] uw.edu  
@kdillmcfarland
